### PR TITLE
Implemented instructions and base app

### PR DESCRIPTION
**Pull request by jordanhudgens**
@AnnJohn Let me know how you'd like me to submit the solution branch, you can see it [here](https://github.com/jordanhudgens/rails-form_for-lab/tree/solution)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
-## Notes
 
- - use form_for to build a new_form.html.erb and an edit_form.html.erb for Student and Class, 4 forms in total.
- - We've given a Student model and a Class model with name.
- - There are two controllers for students and classes with index and show, you need to build new, create, edit, and update.
- - you'll need to create the correct routes with the correct names for form_for to get the route urls correctly.
- - make sure to use form_for.
+## Instructions
+
+For this lab you will be implementing all of the functionality from scratch, including building out the models, views, and controllers for the `SchoolClass` and `Student` resources.
```
Comment (jmburges) [inline README.md]: Give a high level overview of what they'll be building. Like what is the app.

File: spec/factories/school_classes.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
```
Comment (jmburges) [inline spec/factories/school_classes.rb]: No Factories

**Comment by jmburges** [comment]
back at you

**Comment by jmburges** [comment]
disable strong params!

**Comment by jordanhudgens** [comment]
@jmburges I updated the content, removed FG and disabled strong params, I also updated the [solution](https://github.com/jordanhudgens/rails-form_for-lab/tree/solution) branch and ensured all of the tests are still passing there.

**Pull request merged**